TicTacToeClassLibrary/TicTacToe.cs

```diff
@@ -77,20 +84,34 @@ public void Play()
                 }
                 if (IsRoundEnd())
                 {
-                    Console.WriteLine("Do you want to play again? (y/n)");
-                    if (Console.ReadLine() == "y")
-                    {
-                        TicTacToeBoard = Create();
-                        PrintGameInfo();
-                        continue;
-                    }
-                    Console.Clear();
-                    Player1.PrintPlayerInfo();
-                    Player2.PrintPlayerInfo();
-                    break;
+                    HandleRoundEnd();
+                    if (!AskForNewGame())
+                        break;
                 }
             }
         }
+        private void HandleRoundEnd()
+        {
+            Console.WriteLine("Do you want to play again? (y/n)");
+            if (Console.ReadLine() == "y")
+            {
+                TicTacToeBoard = Create();
+                PrintGameInfo();
+            }
+            else
+            {
+                Console.Clear();
+                Player1.PrintPlayerInfo();
+                Player2.PrintPlayerInfo();
+            }
+        }
+        private bool AskForNewGame()
+        {
+            Console.Clear();
+            TicTacToeBoard = Create();
+            PrintGameInfo();
+            return true;
```

Comment:
The game does not end when you press "n". This method always returns true

Reply:
Fixed a bug where entering ‘n’ would result in true, the original version was more rational.